src/containers/FilterNav/index.js

```diff
@@ -80,14 +89,16 @@ const FilterNavComponent = props => {
     { value: 'deadline', label: 'Expiry' }
   ];
 
-  const platforms = reduce(
-    (acc, key) => {
-      acc[key] = { name: key };
-      return acc;
-    },
-    {},
-    appConfig.platform.split(',')
-  );
+  // generates object for SearchSelect component
+  // const platforms = reduce(
+  //   (acc, key) => {
+  //     acc[key] = { name: key };
+  //     return acc;
+  //   },
+  //   {},
+  //   appConfig.platform.split(',')
+  // );
```

Comment:
i kept this code because we'll probably need it in the not too distant future, but let me know if you'd rather just delete it